mmdet/core/export/pytorch2onnx.py

```diff
@@ -118,12 +118,14 @@ def preprocess_example_input(input_config):
     input_path = input_config['input_path']
     input_shape = input_config['input_shape']
     one_img = mmcv.imread(input_path)
+    one_img = mmcv.imresize(one_img, input_shape[2:][::-1])
+    show_img = one_img.copy()
```

Comment:
`show_img` is not normalized while `one_img` is normalized. And line 139 pass `show_img` for pytorch2onnx function. Is this expected behavior?

Reply:
Yes, `show_img` is for visualization, no need to be normalized.